packages/subgraph/bin/build-subgraph.ts

```diff
@@ -3,12 +3,16 @@ import { stringify as yamlStringify } from "yaml";
 
 import { config } from "./config";
 
+const SUPPORTED_NETWORKS = ["mainnet", "xdai"];
```

Comment:
do thegraph still use `xdai` name? 😳

Reply:
https://thegraph.com/docs/en/developing/supported-networks/

You're right! I'll update that legacy CLI name, thanks for the heads-up!
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/28730db3-8b61-441e-83ee-e0c8598f66bd)